### PR TITLE
add ngn-chassis-components v0.0.36 with npm auro-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/ngnjs/chassis-components
**npm package url:** https://www.npmjs.com/package/ngn-chassis-components
**514 downloads in the last month**
@Amomo could u help me check this PR for #7016 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/ngn-chassis-components/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "chassis",
+    "webcomponent"
+  ],
+  "author": {
+    "name": "Corey Butler",
+    "url": "http://coreybutler.com"
+  },
+  "license": "MIT",
+  "homepage": "http://ngn.js.org",
+  "name": "ngn-chassis-components",
```
Comment (PeterDaveHello) [inline ajax/libs/ngn-chassis-components/package.json]: double `"name"`?

**Comment by maruilian11** [comment]
@Amomo i have corrected `npmName`, could you help me check this PR? thanks.

**Comment by PeterDaveHello** [comment]
temp
There are not only 3 minified files in git repo, please confirm that.

**Comment by maruilian11** [comment]
@PeterDaveHello updated with v0.0.36.

**Pull request merged**